documentation/book/kafka-exporter/con-kafka-exporter-lag.adoc

```diff
@@ -29,5 +29,5 @@ Typical actions to reduce lag include:
 * Dropping messages
 
 Actions to reduce consumer lag depend on the underlying infrastructure and the use cases {ProductName} is supporting.
-For instance, the potential impact of lag on broker performance can determine whether consumers are configured to use a disk cache or read from an actual disk.
+For instance, the potential impact of lag on broker performance can determine whether consumers are configured to use a disk cache or read from a physical disk.
```

Comment:
Consumer fetches are not really _configured_ to use disk cache or physical disk. It's just that the kernel is less likely to keep pages for old segments in memory if it's under memory pressure from other consumers.

```suggestion
For instance, a lagging consumer is less likely to benefit from the broker being able to service a fetch request from its disk cache. 
```